README.md

```diff
@@ -87,7 +87,6 @@ More can be found in the docs, but here's a quick list of features which this pr
 - lockScreen()
 - isLocked()
 - shake()
-- complexFind()

```

Comment:
This was removed here
https://github.com/appium/java-client/commit/329e84e801bf9faa9cb2826376ad5959d517d64d